Lib/multiprocessing/context.py

```diff
@@ -53,7 +56,9 @@ def Manager(self):
         can be used to create shared objects.
         '''
         from .managers import SyncManager
-        m = SyncManager(ctx=self.get_context())
+        ctx = self.get_context()
+        m = SyncManager(ctx=ctx)
+        proc_class = ctx.Process
```

Comment:
proc_class isn't used?

Reply:
leftover from an earlier iteration of the code.  removed.